4-routing-in-remix/bee-rich/README.md

```diff
@@ -6,7 +6,7 @@ In this chapter, you will learn about routing in Remix. You will learn how to cr
 
 In the previous chapter you were introduced to BeeRich. You can start right where you left off and use the code from the previous chapter to get started with _Chapter 4, Routing in Remix_.
 
-- If you want to start from scratch, you can find the solution code from the previous chapter [here](../../3-deployment-targets-adapters-and-stacks/bee-rich/).
-- If you are stuck or need a hint, you can check the solution code of this chapter [here](./solution/).
+- If you want to start from scratch, you can find the [solution code from the previous chapter](../../3-deployment-targets-adapters-and-stacks/bee-rich/).
+- If you are stuck or need a hint, you can check the [solution code for this chapter](./solution/).
```

Comment:
If you don't like this style that's totally fine. Can't remember where but I've heard that in general it's good practice to avoid single words like "here" for links. This is mostly an accessibility thing I think, so I don't know how much it'll apply to your book, but just a pattern I've found myself trying to practice more. Basically just link actual phrases or descriptions of where the link goes to.

Reply:
I have heard that as well - also can't remember where though haha thanks for making the change!